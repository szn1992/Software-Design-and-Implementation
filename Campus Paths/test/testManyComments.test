# Test the format when multiple comment and blank lines are entered
# a
# s
#assadada
#sdasdqwdsa
#



#
q