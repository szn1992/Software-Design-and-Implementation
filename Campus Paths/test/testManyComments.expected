Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Test the format when multiple comment and blank lines are entered
# a
# s
#assadada
#sdasdqwdsa
#



#